src/osd/OSD.cc

```diff
@@ -3251,6 +3251,8 @@ struct pistate {
 void OSD::build_past_intervals_parallel()
 {
   map<PG*,pistate> pis;
+  set<PG*> fixed;
```

Comment:
@xiexingguo could you elaborate a little bit the rationale of this change in the commit message?